client/express-checkout/blocks/hooks/use-express-checkout.js

```diff
@@ -58,22 +59,40 @@ export const useExpressCheckout = ( {
 				return;
 			}
 
+			const shippingAddressRequired = shippingData?.needsShipping;
+			const shippingRatesMap = shippingData?.shippingRates[ 0 ]?.shipping_rates?.map(
+				( r ) => {
+					return {
+						id: r.rate_id,
+						amount: parseInt( r.price, 10 ),
+						displayName: r.name,
+					};
+				}
+			);
```

Comment:
```suggestion
			const shippingRatesMap = shippingData?.shippingRates[ 0 ]?.shipping_rates?.map(
				( rate ) => {
					return {
						id: rate.rate_id,
						amount: parseInt( rate.price, 10 ),
						displayName: rate.name,
					};
				}
			);
```
It's not _not_ obvious as currently written and while I think an anonymous function might also be an acceptable exception, I kinda prefer using full-word variable names wherever possible (and coding standards also suggest as much as well). I'm just throwing the suggestion out there, but hardly critical, so feel free to ignore if you'd rather save a few characters instead. 🤷

Reply:
Totally agree with you, I just moved this from the previous implementation without thinking too much about it - thank you for pointing it out!